falcon/request.py

```diff
@@ -293,6 +295,10 @@ def client_accepts_json(self):
         return self.client_accepts('application/json')
 
     @property
+    def client_accepts_jsonapi(self):
```

Comment:
I'm trying to decide how far we should go down the hypermedia path; there are a bunch of media types out there we would have to consider (HAL, Siren, Cj, JSON API, etc.)